examples/ZeroUserCodeLightup.Net6/README.md

```diff
@@ -0,0 +1,87 @@
+# Zero Code light up Example
+
+This example shows how to enable application insights for Kubernetes for ASP.NET Core 6.x WebAPI without code change for the existing project.
+
+## Add Dockerfile
+
+All you need to do is to add a few lines in the [Dockerfile](./dockerfile), and they are:
+
+```dockerfile
+...
+# Adding a reference to hosting startup package
+RUN dotnet add package Microsoft.ApplicationInsights.Kubernetes.HostingStartup --prerelease
+
+# Light up Application Insights for Kubernetes
+ENV APPINSIGHTS_INSTRUMENTATIONKEY $APPINSIGHTS_KEY
+ENV ASPNETCORE_HOSTINGSTARTUPASSEMBLIES Microsoft.ApplicationInsights.Kubernetes.HostingStartup
```

Comment:
```suggestion
ENV ASPNETCORE_HOSTINGSTARTUPASSEMBLIES=Microsoft.ApplicationInsights.Kubernetes.HostingStartup
```
Omitting the equal sign [is not recommended by Docker](https://docs.docker.com/engine/reference/builder/#env)

Reply:
#TIL and fixed.